plugins/integration/providers/voicebase/lib/kVoicebaseFlowManager.php

```diff
@@ -0,0 +1,165 @@
+<?php
+class kVoicebaseFlowManager implements kBatchJobStatusEventConsumer 
+{
+	private $baseEndpointUrl = null;
+	const DEFAULT_ACCURACY = 60;
+	
+	/* (non-PHPdoc)
+	 * @see kBatchJobStatusEventConsumer::shouldConsumeJobStatusEvent()
+	 */
+	public function shouldConsumeJobStatusEvent(BatchJob $dbBatchJob)
+	{
+		$triggerStatuses = array(BatchJob::BATCHJOB_STATUS_DONT_PROCESS, BatchJob::BATCHJOB_STATUS_FINISHED);
+		$jobStatus = $dbBatchJob->getStatus();
```

Comment:
Try to avoid assigning variables in event-consume should methods, these methods called a lot.